### PR TITLE
BC-14790 : Ability to Enable / Disable service like on the console for resource service

### PR DESCRIPTION
**Pull request by OmkeshJosh**

### CODE REVIEW

File: banyan/resource_service_web.go [hunk 1]
```diff
@@ -286,6 +286,12 @@ func WebSchema() (s map[string]*schema.Schema) {
 			Optional:    true,
 			Description: "access tier group which is associated with service",
 		},
+		"enable": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Default:     false,
+			Description: "enable resource service",
```
Comment (rohitSangamnerkar) [inline banyan/resource_service_web.go:293]:
```suggestion
			Description: "enable / disable web service",
```

File: client/service/model.go [hunk 2]
```diff
@@ -24,6 +24,7 @@ type Client interface {
 	Delete(id string) (err error)
 	DetachPolicy(id string) (err error)
 	Disable(id string) (err error)
+	Enabled(id string) (err error)
```
[the remark this answers is not in the document]
Comment (OmkeshJosh) [reply client/service/model.go:27]: Updated

File: banyan/resource_service_web.go (hunk 1 above)
Comment (OmkeshJosh) [reply banyan/resource_service_web.go:293]: Updated

File: banyan/resource_service_web.go [hunk 3]
```diff
@@ -286,6 +286,12 @@ func WebSchema() (s map[string]*schema.Schema) {
 			Optional:    true,
 			Description: "access tier group which is associated with service",
 		},
+		"enable": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Default:     false,
```
Comment (rohitSangamnerkar) [inline banyan/resource_service_web.go:292]: default value for enable needs to be true

**Review by nareshkakubal — APPROVED** [review]

**Pull request merged**